test_conformance/commonfns/test_clamp.cpp

```diff
@@ -169,6 +199,17 @@ int test_clamp_fn(cl_device_id device, cl_context context,
             input_ptr[2][j] = get_random_double(input_ptr[1][j], 0x20000000, d);
         }
     }
+    else if (std::is_same<T, half>::value)
+    {
+        const float fval = 0x200000;
```

Comment:
Thanks; the input values look more useful now.  Using `CL_HALF_MAX` here sounds reasonable to me.

It would indeed be nice to test infinity cases too, but obtaining them from random generation feels a bit unreliable. We could add some special cases in the future perhaps (but that's separate work).  For now, this looks good enough to me.